decidim-elections/app/controllers/decidim/votings/admin/votings_controller.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module Admin
+      # This controller allows the create or update an election.
+      class VotingsController < Admin::ApplicationController
+        include Decidim::Votings::Admin::Filterable
+        helper_method :votings, :voting
+
+        def index
+          enforce_permission_to :read, :votings
+          @votings = filtered_collection
+        end
+
+        def new
+          enforce_permission_to :create, :voting
+          @form = form(VotingForm).instance
+        end
+
+        def create
+          enforce_permission_to :create, :voting
+          @form = form(VotingForm).from_params(params)
+
+          CreateVoting.call(@form) do
+            on(:ok) do
+              flash[:notice] = I18n.t("votings.create.success", scope: "decidim.votings.admin")
+              redirect_to votings_path
+            end
+
+            on(:invalid) do
+              flash.now[:alert] = I18n.t("votings.create.invalid", scope: "decidim.votings.admin")
+              render action: "new"
+            end
+          end
+        end
+
+        def edit
+          # enforce_permission_to :update, :voting, voting: voting
```

Comment:
I guess that we leave this commented code because we are going to use it in the next PR. But I guess that rubocop will complain about it. If not, maybe we forgot to remove it some comment in the following PRs. So, maybe adding a commit removing all that code commented could be an easy way to leave the code clean and be able to easily recover it in the future. What do you think?

Reply:
What your'e saying makes sense, I'm just afraid that, since most of the comments have been already addressed in https://github.com/decidim/decidim/pull/7160, which is built on top of this, following your advice will lead to a conflict nightmare.
If you don't mind, I'll keep this comment as a best practice for the future but won't follow it in this specific case - will take extra care to remove those comments in the mentioned PR :)